downstream/modules/platform/con-hs-eda-sizing-scaling.adoc

```diff
@@ -0,0 +1,11 @@
+[id="con-hs-eda-sizing-scaling"]
+
+= Sizing and scaling guidelines
+
+API nodes process user requests (interactions with the UI or API) while worker nodes process the activations and other background tasks required for Event-Driven Ansible to function properly. The amount of API nodes you require correlates to the desired number of users of the application and the number of worker nodes correlates to the desired number of activations you want to run.
```

Comment:
Add the EDA attribute here. Also, I would say "The number of API nodes" instead of "The amount of API nodes."